coops-ui/src/components/Dashboard/TodoList.vue

```diff
@@ -109,90 +109,88 @@
             </div>
 
             <div class="list-item__actions">
-              <v-col>
```

Comment:
Does the Annual report due date ( BCOMPs exclusive ) line up with the buttons after removing the `<v-col>`?

Reply:
I'll check that...

Reply:
Yes, I think I broke that. I will re-engineer this so the Next AR Date displays above the File Now button. Stand by...

Reply:
Fixed. I replaced the incorrect use of v-col, which came with the additional padding, with a div with the same "width:100%" property.

Reply:
Sounds good!